### ISSUE TITLE
Moon glare often doesn't match where the moon is

### ISSUE THREAD

**Issue by the-r3dacted** [FlashyReese/nuit-interop#6]
![2023-02-04_18 47 07](https://user-images.githubusercontent.com/77039240/216796591-931fe5b3-9adc-4159-841f-25c6b68ee34b.png)
Also since you'll probably need it here's my pack for the sky
https://files.catbox.moe/9lf2az.zip

Sky looks fine with 1.12.2, shaders are Sildurs Vibrant Shaders.

**Comment by FlashyReese** [comment]
While looking at your resource pack's contents, it doesn't contain the necessary textures for conversion.
![image](https://github.com/FlashyReese/fabricskyboxes-interop/assets/30311066/c97ed1ca-8de1-47ab-be1b-9ee46f6a23aa)

Example: `assets/minecraft/mcpatcher/sky/world0/sky_starfield.png` is invalid path so is `assets/minecraft/optifine/terrain/sky0/sky_sunflare.png`/`assets/minecraft/mcpatcher/terrain/sky0/sky_sunflare.png`

Although I tested this on `1.3.3`, it's shouldn't matter much.

If you can provide a updated version of your pack so I can debug the issue because as of right now there isn't anything I can debug if the textures aren't there.

**Comment by the-r3dacted** [comment]
Yeah, I would like a updated pack. I actually stole the sky from the DatOwl pack, which appears to be gone so I have no idea if those files were in that pack or not (iirc it's from 1.8 era).

Edit: Found it on an old drive, here's the original DatOwl pack. https://mega.nz/file/OMcSWKKb#EguYh-2EcH9JNih0u0Snc6MUz57GfJ3KEO8DL6cr5wA
